test/general/__init__.py

```diff
@@ -1,13 +1,15 @@
 from argparse import Namespace
-from typing import Type, Tuple
+from typing import Optional, Tuple, Type
 
 from BaseClasses import MultiWorld, CollectionState
 from worlds.AutoWorld import call_all, World
 
 gen_steps = ("generate_early", "create_regions", "create_items", "set_rules", "generate_basic", "pre_fill")
 
 
-def setup_solo_multiworld(world_type: Type[World], steps: Tuple[str, ...] = gen_steps) -> MultiWorld:
+def setup_solo_multiworld(
+    world_type: Type[World], steps: Tuple[str, ...] = gen_steps, seed: Optional[int] = None
+) -> MultiWorld:
```

Comment:
Did you forget to use seed in set_seed() below?

Reply:
~~no i just really hate the way dlcq and sdv do their tests~~